### ISSUE TITLE
Training of NER model

### ISSUE THREAD

**Issue by stefan-it** [alexandrainst/danlp#6]
Hi,

thanks for open sourcing that library for Danish :heart: 

I've one question about the training of the NER model with multilingual BERT. I just found the benchmark section [here](https://github.com/alexandrainst/danlp/blob/master/docs/models/ner.md#-benchmarks) that uses embeddings from a multilingual BERT model. How was that model trained 🤔

I did some fine-tuning experiments with the upcoming NER training code in (PyTorch-)[Transformers](https://github.com/huggingface/transformers) with [this](https://github.com/huggingface/transformers/pull/1275) implementation and benchmarked results on 5 runs (both development and test set):

| Model | Run 1 | Run 2 | Run 3 | Run 4 | Run 5 | Avg.
| -- | -- | -- | -- | -- | -- | --
| mBERT base, cased (Dev) | 81.48 | 81.43 | 80.55 | 81.37 | 78.5 | 80.666
| mBERT base, cased (Test) | 86.05 | 88.52 | 85.92 | 87.32 | 87.75 | 87.112

I used the DDT dataset from:

```bash
https://danlp.s3.eu-central-1.amazonaws.com/datasets/ddt.zip
```

On the test set the performance difference is ~7% compared to the model, mentioned in the benchmark section. So I was just wondering how that model was trained 🤔

I could of course share the fine-tuned model if you're interested :)

**Comment by hvingelby** [comment]
Hi @stefan-it,

Thank you for checking out the repository! 

The multilingual BERT that we have reported has been trained by [MIPT](https://mipt.ru/english/) and it is a part of the [DeepPavlov](http://docs.deeppavlov.ai/en/master/features/models/ner.html) library. The full details are available [here](https://github.com/deepmipt/DeepPavlov/blob/0.6.1/deeppavlov/configs/ner/ner_ontonotes_bert_mult.json). The model is finetuned on the OntoNotes dataset (in english), which has a larger tagset than the one used in our DDT annotations.

It looks very nice with the performance that you report. Did you simply finetune the multilingual BERT on the train set? It would be nice if you would share the model :smile:

**Comment by stefan-it** [comment]
Thanks for your explanation! I just used the original training data (no additional data) and fine-tuned the multilingual (cased) model of BERT.

I uploaded this fine-tuned model here:

* `config.json`: `wget https://schweter.eu/cloud/bert-base-multilingual-danish-cased/config.json`
* `vocab.txt`: `wget https://schweter.eu/cloud/bert-base-multilingual-danish-cased/vocab.txt`
* `pytorch_model.bin`: `wget https://schweter.eu/cloud/bert-base-multilingual-danish-cased/pytorch_model.bin`

I did use a max sequence length of 128 subwords. Longer sentences were split into smaller ones :)
